doc/ug/conf.py

```diff
@@ -12,6 +12,16 @@
 # serve to show the default.
 
 import sys, os
+import datetime
+
+# Gets version directly from the code
+try:
+    import espressopp
```

Comment:
Would it be possible? The version is hardcoded in Version.hpp so some sort of parsing would be required. Other thing is that, during the building of documentation, ```espressopp``` module has to be available as ```autodoc``` depends on it.